content/en/docs/reference/glossary/kops.md

```diff
@@ -1,32 +1,29 @@
 ---
-title: Kops
+title: kOps (Kubernetes Operations)
 id: kops
 date: 2018-04-12
-full_link: /docs/getting-started-guides/kops/
+full_link: /docs/setup/production-environment/kops/
 short_description: >
-  A CLI tool that helps you create, destroy, upgrade and maintain production-grade, highly available, Kubernetes clusters.
+  kops will not only help you create, destroy, upgrade and maintain production-grade, highly available, Kubernetes cluster, but it will also provision the necessary cloud infrastructure.
 
 aka: 
 tags:
 - tool
 - operation
 ---
- A CLI tool that helps you create, destroy, upgrade and maintain production-grade, highly available, Kubernetes clusters.
+
+`kops` will not only help you create, destroy, upgrade and maintain production-grade, highly available, Kubernetes cluster, but it will also provision the necessary cloud infrastructure.
 
 <!--more--> 
 
 {{< note >}}
-kops has general availability support only for AWS.
-Support for using kops with GCE and VMware vSphere are in alpha.
+AWS (Amazon Web Services) is currently officially supported, with DigitalOcean, GCE and OpenStack in beta support, and Azure in alpha.
 {{< /note >}}
 
-`kops` provisions your cluster with&#58;
-
+kops is an automated provisioning system:
   * Fully automated installation
-  * DNS-based cluster identification
-  * Self-healing&#58; everything runs in Auto-Scaling Groups
-  * Limited OS support (Debian preferred, Ubuntu 16.04 supported, early support for CentOS & RHEL)
-  * High availability (HA) support
-  * The ability to directly provision, or to generate Terraform manifests
-
-You can also build your own cluster using {{< glossary_tooltip term_id="kubeadm" >}} as a building block. `kops` builds on the kubeadm work.
+  * Uses DNS to identify clusters
+  * Self-healing: everything runs in Auto-Scaling Groups
+  * Multiple OS support (Debian, Ubuntu 16.04 supported, CentOS & RHEL, Amazon Linux and CoreOS)
```

Comment:
I'd suggest removing CoreOS. It was end of life in 2020: https://thenewstack.io/say-goodbye-to-coreos/

Reply:
@chrisnegus 
Thank you for your suggestion, I checked the information and saw other information,as follows:

1. **Add OS support**,see the [images.md](https://github.com/kubernetes/kops/blob/master/docs/operations/images.md):
   - Flatcar: Complete support for Flatcar in [kOps 1.17](https://kops.sigs.k8s.io/releases/1.17-notes/)
     - Support Flatcar: https://github.com/kubernetes/kops/pull/7545
   - Rocky: Add support for Rocky in [kOps 1.23](https://kops.sigs.k8s.io/releases/1.23-notes/)
     - Add support Rocky: https://github.com/kubernetes/kops/pull/13559
2. **Remove OS support**,see the [images.md](https://github.com/kubernetes/kops/blob/master/docs/operations/images.md):
   - CentOS: Remove support for CentOS in [kOps 1.23](https://kops.sigs.k8s.io/releases/1.23-notes/)
     - Remove CentOS: https://github.com/kubernetes/kops/pull/12877
   - CoreOS: Remove support for CoreOS in [kOps 1.19](https://kops.sigs.k8s.io/releases/1.19-notes/)
     - Remove CoreOS: https://github.com/kubernetes/kops/pull/9065
     - [CoreOS EOL](https://thenewstack.io/say-goodbye-to-coreos/)
 3.  All kOps OS support can see the [images.md](https://github.com/kubernetes/kops/blob/master/docs/operations/images.md).

Reply:
I submit a PR(https://github.com/kubernetes/website/pull/36796) to update.

Reply:
Very nice. Well done.